### ISSUE TITLE
[spec bug] DynamoDB BillingMode/PointInTimeRecoverySpecification in some regions

### ISSUE THREAD

**Issue by PatMyron** [aws-cloudformation/cfn-python-lint#1065]
DynamoDB `BillingMode`/`PointInTimeRecoverySpecification` are only available in the regions in the region dropdown on these pages:
https://aws.amazon.com/dynamodb/pricing/on-demand/
https://aws.amazon.com/dynamodb/pricing/provisioned/

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Resources: 
  dynamoDBTable: 
    Type: AWS::DynamoDB::Table
    Properties: 
      AttributeDefinitions: 
        - AttributeName: "Album"
          AttributeType: "S"
        - AttributeName: "Artist"
          AttributeType: "S"
      KeySchema: 
        - AttributeName: "Album"
          KeyType: "HASH"
        - AttributeName: "Artist"
          KeyType: "RANGE"
      BillingMode: PAY_PER_REQUEST
      PointInTimeRecoverySpecification:
        PointInTimeRecoveryEnabled: true
```

`One or more parameter values were invalid: Unsupported input parameter BillingMode (Service: AmazonDynamoDBv2; Status Code: 400; Error Code: ValidationException; Request ID:)`

`Continuous Backups has not been enabled for DynamoDB table yet`

---

only verified in `ap-northeast-3`, but might affect more regions like `us-gov-east-1`

**Comment by PatMyron** [comment]
While we patch [Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) regional availability automatically from [SSM](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/scripts/update_specs_services_from_ssm.py) at the resource type level and [manually](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/src/cfnlint/data/ExtendedSpecs/ap-south-1/01_spec_patch.json), I don't think we can commit to maintaining manual regional availability patches down to the property type level with ~1500 and growing property types and ~30 and growing AWS regions
